DFC.App.DiscoverSkillsCareers.Core/Extensions/HttpRequestExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using DFC.App.DiscoverSkillsCareers.Core.Constants;
+using Microsoft.AspNetCore.Http;
+
+namespace DFC.App.DiscoverSkillsCareers.Core.Extensions
+{
+    public static class HttpRequestExtensions
+    {
+        public static bool IsRequestFromComposite(this HttpRequest httpRequest)
+        {
+            var result = false;
+            if (httpRequest != null && httpRequest.Headers != null)
+            {
+                result = httpRequest.Headers.ContainsKey(HeaderName.CompositeRequest);
```

Comment:
What happens if this can't be hit or it crashes for any reason?

Reply:
Its expected that this will be run in the context of a web application